### PR TITLE
fix(PX-3984): add background for the footer 

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
This PR adds white background to the footer to fix the partner profile page. Also adds `AppContainer` to set to the footer the same width as for the page (please let me know if this is unnecessary).

JIRA - https://artsyproduct.atlassian.net/browse/PX-3984

Before:
image

Now:
image

ezgif com-gif-maker (14)

### CODE REVIEW

**Review by The-Beez-Kneez — APPROVED** [review]

**Review by damassi — COMMENTED** [review]
👀

File: src/v2/Apps/Components/AppShell.tsx [hunk 1]
```diff
@@ -66,7 +67,12 @@ export const AppShell: React.FC<AppShellProps> = props => {
       </Box>
 
       <NetworkOfflineMonitor />
-      <HorizontalPadding>{showFooter && <Footer />}</HorizontalPadding>
+
+      <Flex backgroundColor="white">
```
Comment (damassi) [inline src/v2/Apps/Components/AppShell.tsx:71]: This should match the white color in our theme: https://github.com/artsy/palette/blob/master/packages/palette/src/themes/v2.tsx#L113

Comment (damassi) [inline src/v2/Apps/Components/AppShell.tsx:71]:
```suggestion
      <Flex backgroundColor="white100">
```

**Review by damassi — APPROVED** [review]
Added one tiny change so that the color matched whats in [our theme](https://github.com/artsy/palette/blob/master/packages/palette/src/themes/v2.tsx#L113), but otherwise pulled things down and all looks good 👍

**Comment by damassi** [comment]
#mergeOnGreen

**Pull request merged**